locales/en-US/snoozetabs.properties

```diff
@@ -0,0 +1,45 @@
+extDesc=An add-on to let you snooze your tabs for a while.
```

Comment:
I think the concept of 'snooze' is going to be hard to grasp for some localizers. Can we add a really short comment explaining what the add-on does? That will help them find a good translation.

To give you an idea, there's no perfect translation of 'snooze' in Italian, it's commonly translated as 'pospone'.